tools/infrasys/src/root.rs

```diff
@@ -48,22 +60,26 @@ pub fn create_root(root_role_path: &Path) -> Result<()> {
         thing: "root role",
     })?;
     fs::create_dir_all(role_dir).context(error::MkdirSnafu { path: role_dir })?;
+
+    let default_region = thread::spawn(move || {
```

Comment:
Is there any reason to do this async?  Can we do it synchronously?  Spinning up runtimes in the middle of a program can get messy if other places are doing the same.